app/components/Entity/components/ConfigureWorkflowInputs/components/Main/components/Stepper/components/Step/hooks/UseLaunchGalaxy/useLaunchGalaxy.ts

```diff
@@ -38,7 +41,8 @@ export const useLaunchGalaxy = ({
               configuredValue.geneModelUrl,
               configuredValue.readRunsSingle,
               configuredValue.readRunsPaired,
-              workflow.parameters
+              workflow.parameters,
+              config?.browserURL
```

Comment:
Is that necessary for typing ? Would
```suggestion
              config.browserURL
```
work ?